enhancements/declarative-index-config.md

```diff
@@ -857,178 +863,7 @@ $ cat community-operators/etcd/etcd.json
         }
     ]
 }
-$ opm index add --bundles quay.io/operatorhubio/etcd:v0.9.0 --mode replaces --tag docker.io/my-namespace/community-operators:latest
```

Comment:
Is there a reason we removed this blob of JSON? I think we still want to have a section somewhere about what the JSON schema looks like. We do intend to let users manually modify this JSON blob right?

Reply:
It is there here https://github.com/operator-framework/enhancements/pull/80/files#diff-1edbfba9fdddd2b74b5cd32c6de146a4f62e1400a93bee78aed2cbf6ebaaa1d7R867-R971

This edit is just removing it from `opm index add` and `opm registry add`